src/locales/fr-FR/admin.ftl

```diff
@@ -77,9 +77,9 @@ configure-sideBarNavigation-moderation = Modération
 configure-sideBarNavigation-organization = Organisation
 configure-sideBarNavigation-advanced = Avancé
 configure-sideBarNavigation-email = Email
-configure-sideBarNavigation-bannedAndSuspectWords = Mots bannis et suspects
+configure-sideBarNavigation-bannedAndSuspectWords = Mots bannis
```

Comment:
banned and suspect are distinct concepts, this headline is above both I believe.

Reply:
yeah, my mystake, sorry.